demo-edm-dexed/b01-index.js

```diff
@@ -0,0 +1,52 @@
+const fs      = require('fs');
```

Comment:
Is this an updated version of b00-index.js? If so, remove the obsolete version, and rename this one to `index.js` otherwise, document how this one is different.